vllm/model_executor/models/llama_eagle.py

```diff
@@ -22,6 +23,23 @@
 
 logger = init_logger(__name__)
 
+# Weight name mapping for speculators format compatibility
+SPECULATORS_WEIGHT_MAP = {
+    "fusion_fc.weight": "fc.weight",
```

Comment:
I though we removed this translation in speculators?

Reply:
we haven't so far, there was contention behind eagle1 naming, we will remove the need for this translation in a follow up PR